### PR TITLE
Create an event from EventListView

### PR DESCRIPTION
**Pull request by misstick**
- [x] make popover as generic as possible (to be used from several kind of views),
 - [x] add a button on `EventListView` to add an event,
 - [x] display popover into `EventListView`,
 - [x] fix popover display into mobile screens.

### CODE REVIEW

**Comment by misstick** [comment]
@jsilvestre : I'm waiting your approval to merge this PR

File: client/app/router.coffee [hunk 1]
```diff
@@ -107,10 +117,63 @@ module.exports = class Router extends Backbone.Router
...
+
         app.menu.activate 'calendar'
         @onCalendar = true
 
 
+    showPopover: (options={}) =>
+        # @TODO Event creation is a typical core feature of the calendar
+        # app, this part should be moved directly into the app module, and
+        # managed
+        # with event handlers
```
Comment (aenario) [inline client/app/router.coffee]: Not TODO anymore :clap:

**Comment by jsilvestre** [comment]
@aenario thanks for the review. I validated the feature, so I'll add it to the support lv3 board so you can release next week. Is it ok with you?

**Comment by aenario** [comment]
ok

**Pull request merged**